taskcluster/docker/linux/Dockerfile

```diff
@@ -159,7 +159,7 @@ ENV CCACHE=sccache \
 RUN \
     curl -sfSL --retry 5 --retry-delay 10 \
          -o /usr/local/bin/tooltool.py \
-         https://raw.githubusercontent.com/mozilla/build-tooltool/36511dae0ead6848017e2d569b1f6f1b36984d40/tooltool.py && \
+         https://raw.githubusercontent.com/mozilla-releng/tooltool/master/client/tooltool.py && \
```

Comment:
I see `tooltool` has since changed homes - I imagine the `master` branch is stable enough for us not to fix a tag/revision? +1 for this because it allows us to catch and report any oddities with the script quickly - just curious

Reply:
Yeah, my reasoning was that it changes infrequently enough to not be an issue. Especially since we're only going to be attempting to pick up a newer version if/when the docker image rebuilds, which should be a pretty infrequent occurrence.